src/Altinn.Profile/Controllers/UsersController.cs

```diff
@@ -58,11 +58,11 @@ public async Task<ActionResult<UserProfile>> Get(int userID)
         /// </summary>
         /// <param name="userUuid">The user uuid</param>
         /// <returns>The information about a given user</returns>
-        [HttpGet("{userUuid:Guid}")]
+        [HttpGet("")]
         [Authorize(Policy = "PlatformAccess")]
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public async Task<ActionResult<UserProfile>> Get(Guid userUuid)
+        public async Task<ActionResult<UserProfile>> Get([FromQuery] Guid userUuid)
```

Comment:
Do we know if this is going to be enough to make API Management happy? The alternative would be to add something unique to the path. Not ideal, but an option.

Reply:
I do not know will add something unique to path